airbyte-ci/connectors/connector_ops/connector_ops/acceptance_test_config_checks.py

```diff
@@ -9,13 +9,33 @@
 
 from connector_ops import utils
 
-RELEASE_STAGE_TO_STRICTNESS_LEVEL_MAPPING = {"generally_available": "high"}
+HIGH_TEST_STRICTNESS_LEVEL_THRESHOLD = {
+    "sl": 300,
+    "ql": 400,
+}
 BACKWARD_COMPATIBILITY_REVIEWERS = {"connector-operations", "connector-extensibility"}
 TEST_STRICTNESS_LEVEL_REVIEWERS = {"connector-operations"}
 GA_BYPASS_REASON_REVIEWERS = {"connector-operations"}
 GA_CONNECTOR_REVIEWERS = {"gl-python"}
 REVIEW_REQUIREMENTS_FILE_PATH = ".github/connector_org_review_requirements.yaml"
 
+def is_important_connector(connector: utils.Connector) -> bool:
```

Comment:
Opted to use the idea of important connector which is either a high sl and/or high ql. 

This is because we may have a connector that is deemed qualitatively important (SL) and we should enable high strictness in advance of a high QL.

defn: https://docs.google.com/document/d/161FF__wOjwS6j-1fVyHexDD1cJCQ5RJ2LMC8FI2N6yI/edit

The alternative is to use just QL.